### PR TITLE
fix(telemetry): wrong average volume sizes due to V2 volume counts

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/7380

### CODE REVIEW

File: controller/setting_controller.go [hunk 1]
```diff
@@ -1541,6 +1541,7 @@ func (info *ClusterInfo) collectVolumesInfo() error {
 		return errors.Wrapf(err, "failed to list Longhorn Volumes")
 	}
 	volumeCount := len(volumesRO)
+	volumeCountV1 := types.GetVolumeCountByBackendStoreDriver(longhorn.BackendStoreDriverTypeV1, volumesRO)
```
Comment (innobead) [inline controller/setting_controller.go:1544]: I think `GetVolumeCountByBackendStoreDriver` is very specific for this usage and would not be used later, suggest adding here as a private function instead.

P.S. If needed, we can move it to a shared package later.

Or add `FilterVolumesByBackendStoreDriver` instead, which would be potentially reusable.. 

```go
volumeCountV1 := len(types.FilterVolumesByBackendStoreDriver(volumes, longhorn.BackendStoreDriverTypeV1))
```

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, but just one comment needed to resolve.

File: controller/setting_controller.go (hunk 1 above)
Comment (c3y1huang) [reply controller/setting_controller.go]: I prefer removing the function as there are no other uses for either one at this point, and the logic is straightforward.

**Review by innobead — APPROVED** [review]

**Comment by c3y1huang** [comment]
@mergify backport v1.5.x

**Pull request merged**